src/index.js

```diff
@@ -19,21 +19,24 @@ import Round from './Round';
 import Turn from './Turn';
 import domUpdates from './domUpdates.js';
 
-var data;
+var boards;
```

Comment:
This global variable is used the make the API data accessible outside the fetch function. Best practice? Hacky?

Reply:
I’d probably store this data as an instance property (maybe on the Game class) just to avoid using a global. If you set this as a global in your app code, your tests might have a hard time not recognizing it. You could also leave it as-is and try to set a global in your test file like `global.boards = <yourmockdataset>`.